cmd/controller-manager/main.go

```diff
@@ -94,6 +98,8 @@ func main() {
 
 	glog.Infof("Federation namespace: %s", fedNamespace)
 	glog.Infof("Cluster registry namespace: %s", clusterNamespace)
+	glog.Infof("Cluster available delay: %d", clusterAvailableDelay)
```

Comment:
Do these need to be logged?  The namespaces are logged for troubleshooting purposes since they may be set from either args or metadata and getting it wrong will render the control plane unusable.

Reply:
+1 to have the log here, as it will help for debugging.